dags/copy_deduplicate.py

```diff
@@ -105,6 +105,21 @@
         ],
     )
 
+    with TaskGroup(group_id='main_ping_downstream_external') as main_ping_downstream_external:
```

Comment:
Could be more succinct.
```suggestion
    with TaskGroup('main_ping_downstream_external') as main_ping_downstream_external:
```